### ISSUE TITLE
Denoise

### ISSUE THREAD

**Issue by BlackBaiX** [lipengFu/IRCNN#1]
How do I modify the model if I want the model input to be noise image and output to be denoise image?

**Comment by Obliviously** [comment]
First off: You wouldn't necessary change the model to do that. Secondly the program is already doing that as far as i can see.

**Comment by anjingxing** [comment]
@lipengFu thank you very much for sharing the code. 
May I ask what is the requirement of the data set? Can you share the data set you adopted?

**Comment by lipengFu** [comment]
@anjingxing 时间有点久了，数据集应该是BSD500中的图像

**Comment by anjingxing** [comment]
@lipengFu thank you，As a beginner,I don't know a lot about it，I'm sorry to disturb you。

**Comment by lipengFu** [comment]
@anjingxing http://www.eecs.berkeley.edu/Research/Projects/CS/vision/grouping/BSR/BSR_bsds500.tgz

**Comment by anjingxing** [comment]
@lipengFu thank you   very much

**Comment by deepLearnerNew** [comment]
@anjingxing did you happen to run the code? Can you please share which data set you used and how to train and test?

@lipengFu If any help from you will be really great.
Hope I am not asking for much.

**Comment by ACEaaaaa** [comment]
请问下，如何处理训练数据集让代码跑起来

**Comment by navy63** [comment]
这个代码貌似存在问题
 self.conv = slim.conv2d(conv, 3, [3, 3], rate=self.dilate[6], activation_fn=None, weights_regularizer=slim.l2_regularizer(self.hps.weight_decay_rate), scope='conv_%d'%(self.hps.num_conv), reuse=self.reuse)
self.clear = self._image - self.conv
那么问题来了，计算损失的时候用self.conv 这个应该是噪声图像，计算结果的时候用model.clear，自相矛盾
content_cost = (1./self.hps.batch_size)*tf.nn.l2_loss(self.conv - self.label)
tf_psnr = tf.image.psnr(labels, model.clear, 1.)
